challenges/CC_61_fractal_spirograph/CC_61_fractal_spirograph.pde

```diff
@@ -10,12 +11,13 @@ int resolution = 50;
 
 Orbit sun;
 Orbit end;
+Orbit next;
 
 void setup() {
   size(600, 600);
   path = new ArrayList<PVector>();
   sun = new Orbit(width/2, height/2, width/4, 0);
-  Orbit next = sun;
+  next = sun;
```

Comment:
I think I prefer to leave next as a local variable for the loop only? It's comparable to `int i = 0` in a for loop. Is that ok with you?